pkg/kubeadm/api/types.go

```diff
@@ -37,8 +37,12 @@ type KubeadmConfig struct {
 
 type InitFlags struct {
 	API struct {
-		AdvertiseAddrs  []net.IP
-		ExternalDNSName []string
+		AdvertiseAddrs        []net.IP
+		ExternalDNSName       []string
+		ExternalEtcdEndpoints []string
```

Comment:
I'm trying to avoid over-engineering stuff because I'm seeing this going to a configuration file in the future rather than relying on flags.